src/main/java/application/RacingGameApplication.java

```diff
@@ -0,0 +1,59 @@
+package application;
+
+import domain.Car;
+import domain.Cars;
+import utils.NumberGenerator;
+import domain.TryCount;
+import dto.request.CarNameDto;
+import dto.response.CarStatusDto;
+import dto.response.WinnersNameDto;
+import view.InputView;
+import view.OutputView;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingGameApplication {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final NumberGenerator numberGenerator;
+
+    public RacingGameApplication(InputView inputView, OutputView outputView, NumberGenerator numberGenerator) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.numberGenerator = numberGenerator;
+    }
+
+    public void run() {
+        List<CarNameDto> carNameDtos = inputView.sendCarsName();
+        TryCount tryCount = inputView.sendTryCount();
+
+        Cars cars = getCars(carNameDtos);
+
+        playGame(tryCount, cars);
+        printResult(cars);
+    }
+
+    private Cars getCars(List<CarNameDto> carNameDtos) {
+        return carNameDtos.stream()
+                .map(carNameDto -> new Car(carNameDto.getName()))
+                .collect(Collectors.collectingAndThen(Collectors.toList(), Cars::new));
+    }
+
+    private void playGame(TryCount tryCount, Cars cars) {
+        for (int i = 0; i < tryCount.getTryCount(); i++) {
+            cars.moveCars(numberGenerator);
```

Comment:
```suggestion
            cars.move(numberGenerator);
```

cars.moveCars()는 `cars`가 중복되니까 cars.move()로만 써도 충분하다고 생각해요 ㅎㅎ

Reply:
클래스의 이름이 있어서 동어반복같은 느낌이였네요 !!